mars/dataframe/groupby/aggregation.py

```diff
@@ -825,9 +825,7 @@ def _execute_map(cls, ctx, op: "DataFrameGroupByAgg"):
 
         for input_key, output_key, cols, func in op.pre_funcs:
             if input_key == output_key:
-                ret_map_groupbys[output_key] = (
-                    grouped if cols is None else grouped[cols]
-                )
+                ret_map_groupbys[output_key] = grouped
```

Comment:
This removal disables possible data truncation and may introduce performance issue. Maybe a better solution is needed.